source/beamline/BioXAS/BioXASBeamline.cpp

```diff
@@ -81,8 +81,6 @@ AMAction3* BioXASBeamline::createDarkCurrentMeasurementAction(double dwellSecond
 
 AMAction3* BioXASBeamline::createScanInitializationAction(AMGenericStepScanConfiguration *configuration)
```

Comment:
Same as above, I think this should live BioXAS specified GenericStepScanController.

At this moment, only BioXAS and Ideas implemented this function in the Beamline class